src/NodaTime/Calendars/RegularWeekYearRule.cs

```diff
@@ -70,7 +71,31 @@ private RegularWeekYearRule(int minDaysInFirstWeek)
         /// <returns>A <see cref="RegularWeekYearRule"/> with the specified minimum number of days in the first
         /// week.</returns>
         public static RegularWeekYearRule ForMinDaysInFirstWeek(int minDaysInFirstWeek)
-            => new RegularWeekYearRule(minDaysInFirstWeek);
+            => ForMinDaysInFirstWeek(minDaysInFirstWeek, IsoDayOfWeek.Monday);
+
+        /// <summary>
+        /// Creates a week year rule where the boundary between one week-year and the next
+        /// is parameterized in terms of how many days of the first week of the week
+        /// year have to be in the new calendar year, and also by which day is deemed
+        /// to be the first day of the week.
+        /// </summary>
+        /// <remarks>
+        /// <paramref name="minDaysInFirstWeek"/> determines when the first week of the week-year starts.
+        /// For any given calendar year X, consider the Monday-to-Sunday week that includes the first day of the
```

Comment:
It is for this factory method - because you don't pass in a first day of week. I'll clarify that a bit.